ios/AppLovinMAX.m

```diff
@@ -206,6 +206,128 @@ - (instancetype)init
     }];
 }
 
+#pragma mark - Data Passing
+
+RCT_EXPORT_METHOD(setTargetingDataYearOfBirth:(nonnull NSNumber *)yearOfBirth)
+{
+    if ( !_sdk )
+    {
+        [self logUninitializedAccessError: @"setTargetingDataYearOfBirth"];
+        return;
+    }
+    
+    self.sdk.targetingData.yearOfBirth = yearOfBirth.intValue <= 0 ? nil : yearOfBirth;
+}
+
+RCT_EXPORT_METHOD(setTargetingDataGender:(NSString *)gender)
+{
+    if ( !_sdk )
+    {
+        [self logUninitializedAccessError: @"setTargetingDataGender"];
+        return;
+    }
+    
+    ALGender alGender = ALGenderUnknown;
+
+    if ( [@"F" isEqualToString: gender] )
+    {
+        alGender =  ALGenderFemale;
+    }
+    else if ( [@"M" isEqualToString: gender] )
+    {
+        alGender =  ALGenderMale;
+    }
+    else if ( [@"O" isEqualToString: gender] )
+    {
+        alGender =  ALGenderOther;
+    }
+
+    self.sdk.targetingData.gender = alGender;
+}
+
+RCT_EXPORT_METHOD(setTargetingDataMaximumAdContentRating:(nonnull NSNumber *)maximumAdContentRating)
```

Comment:
We usually assume `nonnull` in our codebase, and only annotate if something is `nullable`.

Reply:
Without nonnull, RN generates an error like:

`Argument 0 (NSNumber) of AppLovinMAX.setTargetingDataMaximumAdContentRating has unspecified nullability but React requires that all NSNumber arguments are explicitly marked as nonnull to ensure compatibility with Android`

We can use ‘double’ for passing numbers in RN but double is not straightforward for comparison.  So I think that nonnull is ok for this case.

Reply:
Ah ok, I remember that being an issue in RN.